pull/context.go

```diff
@@ -233,3 +236,11 @@ type CollaboratorPermission struct {
 	// repository. If false, the permission is granted by the organization.
 	ViaRepo bool
 }
+
+type Body struct {
+	Body         string
+	CreatedAt    time.Time
+	Editor       string
```

Comment:
looks like we don't actually use editor anywhere, i think we were thinking about making a comparison in the ignore edit logic to allow edits to the body if the editor was the same as the the author.

in that case, making candidate type a thing like @bluekeyes suggested is probably worth considering. do you think that's still valuable?